src/env_loader.zig

```diff
@@ -909,58 +534,251 @@ pub const Loader = struct {
     }
 };
 
-pub const Parser = struct {
-    pub fn parse(
-        source: *const logger.Source,
+const Parser = struct {
+    pos: usize = 0,
+    src: string,
+
+    const whitespace_chars = "\t\x0B\x0C \xA0\n\r";
+
+    fn skipLine(this: *Parser) void {
+        var i = this.pos;
+        while (i < this.src.len) : (i += 1) {
```

Comment:
Can this use strings.indexOfNewlineOrNonASCII?

Reply:
(I ask because it uses SIMD)

Reply:
fixed `strings.indexOfAny()`, employed SIMD there and converted this function to call that